frontends/infinite-corridor/src/pages/field-details/EditFieldAppearanceForm.tsx

```diff
@@ -65,7 +65,7 @@ const EditFieldAppearanceForm = (props: FormProps): JSX.Element => {
                 ) : null}
               </div>
             </div>
-            <div className="row form-item">
+            <div className="form-row form-item">
```

Comment:
Our `row` class was specifically for forms, and `row` is hard to search for. So I changed it to `form-row`, which is a little easier.